test/plugin_functional/plugins/search_explorer/public/do_search.tsx

```diff
@@ -24,7 +24,7 @@ import { Observable } from 'rxjs';
 import {
   IKibanaSearchResponse,
   IKibanaSearchRequest,
-} from '../../../../../src/plugins/search/public';
+} from '../../../../../src/plugins/search/common';
```

Comment:
I think this one is actually supposed to be public, not common, because you aren't supposed to be allowed to import from common from another plugin.  I think it's a mistake that this isn't caught by the linter.  I've had chats with Court about this kind of thing though so it may change so that we officially allow it.